doc/classes/VisualScriptSequence.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptSequence" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Executes a series of Sequence ports. 
 	</brief_description>
 	<description>
+		This node steps through a series of one or more output Sequence ports. The [code]current[/code] data port outputs the currently executing item.
```

Comment:
`Steps through a series of one or more output Sequence ports. The [code]current[/code] data port outputs the currently executing item.`
I don't understand what this does or it is for.